src/main/resources/static/css/index.css

```diff
@@ -7,14 +7,7 @@
 
                                                                                        /* FONT */
 
-/*
-P.S.
-Using fibonacci series for font size and line height.
-I thing it looks really good.
-Joe
```

Comment:
Put conversation in the pull request... not in source code comments.  This is worth removing.